editor.planx.uk/src/@planx/components/Pay/Public/FeeBreakdown/FeeBreakdown.tsx

```diff
@@ -98,7 +98,7 @@ const ApplicationFee: FeeBreakdownRow = ({ amount }) => (
 );
 
 const Reductions: FeeBreakdownRow = ({ amount, reductions }) => {
-  if (!amount.reduction) return null;
+  if (reductions.length === 0) return null;
```

Comment:
Reduction & exemption `amount` can legitimately be 0. Previously this meant they wouldn't be shown even if they were picked up, so now we'll check length to ensure if they exist they are displayed, independent of amount